lambdas/src/Server.ts

```diff
@@ -52,16 +52,17 @@ export class Server {
       Metrics.initialize()
     }
 
-    // Base endpoints
-    this.registerRoute('/status', controller, controller.getStatus)
-
     const ensOwnership: EnsOwnership = env.getBean(Bean.ENS_OWNERSHIP)
     const wearablesOwnership: WearablesOwnership = env.getBean(Bean.WEARABLES_OWNERSHIP)
     const fetcher: SmartContentServerFetcher = env.getBean(Bean.SMART_CONTENT_SERVER_FETCHER)
     const contentClient: SmartContentClient = env.getBean(Bean.SMART_CONTENT_SERVER_CLIENT)
     const theGraphClient: TheGraphClient = env.getBean(Bean.THE_GRAPH_CLIENT)
     const offChainManager: OffChainWearablesManager = env.getBean(Bean.OFF_CHAIN_MANAGER)
 
+    // Base endpoints
+    this.registerRoute('/status', controller, controller.getStatus)
+    this.registerRoute('/health', controller, controller.getHealth)
```

Comment:
This is up for discussion, but other than the status endpoint, we've avoiding the controller. We've simply been adding endpoints under `src/apis`. I don't know what the rest of the people think. If we should continue doing that or not